content/docs/getting-started.md

```diff
@@ -36,18 +36,16 @@ React has been designed from the start for gradual adoption, and **you can use a
 
 ### Online Playgrounds
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s/new).
+To start playing with React instantly, you can use an online code playground, without having to bother configuration and environment setup. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s/new).
```

Comment:
Missing "with" after "bother"?

That said I don't think *we* should give people the idea about React needing configuration or environment setup. Let's not forget new readers aren't necessarily expecting this either.